src/stories/ThemeProvider.stories.tsx

```diff
@@ -8,6 +8,7 @@ import {createGlobalStyle} from 'styled-components'
 export default {
   title: 'Generic behaviors/ThemeProvider',
   component: ThemeProvider,
+  parameters: {disableThemeDecorator: true},
```

Comment:
I'm still seeing the the theme switcher in toolbar, but it doesn't do anything. Not a blocker, but it feels a bit strange.

![Screen Shot 2022-04-28 at 4 36 43 PM](https://user-images.githubusercontent.com/2313998/165841331-d356daf2-164d-459a-9d55-ec68e9b52842.png)

Reply:
👀 damn, that's bad. There doesn't seem to be a way for customising the toolbar without writing a custom addon 😓